README.md

```diff
@@ -84,7 +84,14 @@ Then run
 pipenv run python generate_print_files.py <path to config csv> <path to output directory> <batch ID> --no-gcs
 ```
 
-this will read the generated qid/uac pairs and generate the print and manifest files in the specified directory
+This will read the generated qid/uac pairs and generate the print and manifest files in the specified directory
+
+If the print file is for CCS unaddressed questionnaires, then the UAC's generated for the QID's will not be included 
+
+The print files will be encrypted so in order to decrypt them and read the contents, import the key with
```

Comment:
Nice 👍 Might be worth mentioning our dummy password here (it's fine as it's just used for our tests) to save searching the unit tests for the password?